modules/reactor/src/meshgenerators/PeripheralRingMeshGenerator.C

```diff
@@ -1,488 +1,493 @@
-//* This file is part of the MOOSE framework
-//* https://www.mooseframework.org
-//*
-//* All rights reserved, see COPYRIGHT for full restrictions
-//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
-//*
-//* Licensed under LGPL 2.1, please see LICENSE for details
-//* https://www.gnu.org/licenses/lgpl-2.1.html
-
-#include "PeripheralRingMeshGenerator.h"
-
```

Comment:
this file has these line endings now: ^M
https://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it

they ll have to be removed.
they typically happen with opening files in windows

Reply:
Forgot to switch from CRLF to LF. (Is this supposed to be captured by CIVET?)